src/chapters/angular-wk1/components.rst

```diff
@@ -0,0 +1,69 @@
+Components
+===========
+
+.. index:: ! angular-component
+
+In Angular a **component** controls a patch of screen called a view.
+
+Angular builds a web page by combining multiple **components** together. Splitting our page into individual components makes our application more organized, and increases our ability to focus on one section of our web application at a time.
+
+Everything in Angular centers on the idea of building a webpage from separate, smaller pieces. We must understand how to get these pieces to work together, and that begins by exploring what makes up each individual component.t. So it is crucial to understand the individual pieces of a component, and how to get component to work together.
+
+Component Files
+---------------
+
+A component consists of 4 files:
+    
+    #. an HTML file (.html)
+    #. a CSS file (.css)
+    #. a typescript file (.ts)
+    #. a test file (.spec.ts)
+
+.. figure:: ./figures/ComponentPieces.png
+   :alt: Visual of the files associated with a Component.
+
+Looking at the file tree, we see that all four files contain the name of the component. Also, the files are located in a folder named after the component. In the example above the component was named header.
+
+As an example, if we create a new component named task-list our four files would be found in the task-list folder and would be called:
+    #. task-list.html
+    #. task-list.css
+    #. task-list.ts
+    #. task-list.spec.ts
+
+Each file contains the information necessary for ONLY that component. ``task-list.html`` holds the HTML required for the task-list and no other component. ``task-list.css`` only styles task-list, the typescript code in ``task-list.ts`` only applies to this component, and all the tests for this component will be found in ``task-list.spec.ts``.
+
+Adding a New Component
+----------------------
+
+Each component is a smaller part of an overall web application. The main component serves as a base structure, and it comes standard with all Angular applications. It's the container that holds all of the other components, and it organizes them into the web application.
+
+An important thing to understand is how new components are created and then added to the main component.
+
+When you generate a new component using a tool like Stackblitz or the Angular CLI, it is automatically added to the main component. Let's explore how this process works.
```

Comment:
The StackBlitz reference becomes obsolete if we move ahead with adding the terminal, GitBash, and IDE content.